roles/apply-config-update/tasks/beta-3.yaml

```diff
@@ -12,4 +12,17 @@
   when: grep_prater_mapping.stdout == "0"
   become: yes
 
+- name: Test for exit-account
+  shell: grep -c "^\s{4}exit_account:\slighthouse-only" /etc/stereum/ethereum2.yaml || true
+  register: exit_account
+  become: yes
+
+- name: Add exit-account to config
+  lineinfile:
+    dest: "/etc/stereum/ethereum2.yaml"
+    insertafter: "^\s{4}create_account:\slighthouse-only/create-account.yaml"
```

Comment:
that's not gonna work. `/` is a special character for regex, also `.`

you can use the regex I posted earlier or escape the special characters

Reply:
added `\` before special characters. It tested with https://regex101.com/.